### PR TITLE
chore: add cypress 13.5.0

### PR DESCRIPTION
**Pull request by gregmagolan**
Also, re-generate checksums for older versions since the linux checksums have changed

### CODE REVIEW

File: cypress/private/versions.bzl [hunk 1]
```diff
@@ -1,34 +1,41 @@
...
+        "linux-x64": "d1fba34ba408514506c162be8041dfeb37ac42f777ec0e369c58cd0edbcaa000",
+        "linux-arm64": "edd4d76f0ab4228d7207ad57aa4c08a2a60528d056a6ea0397ae3f1c4d365d51",
+        "win32-x64": "35c157342b8ae120e1e37e519d255d60e5bec3634bb9c488cda161ece24cc4e5",
+    },
     "13.3.1": {
         "darwin-x64": "417c5f1d77e15c8aef0a55f155c16c3dbbc637f918c1e51f8fec6eb1c73a9ba9",
         "darwin-arm64": "143d905779c0b0a8a9049b0eb68b4b156db3d838d4546ce5082a8f7bd5dc5232",
-        "linux-x64": "a165aa290d23d03f191852df2e8c3cb25ddccf1d4e226518a0a217f4ec405328",
-        "linux-arm64": "b51810d7fda748f67d5d875d5e265ee22bb394af2bb4a545b78c99f1aae91cb0",
+        "linux-x64": "bb0ddd980bd82792a477b1c39ce8a0a7b275481031c559c065c94f1131151b0c",
```
Comment (jbedard) [inline cypress/private/versions.bzl:16]: Why did these change for old versions?

Comment (gregmagolan) [reply cypress/private/versions.bzl:16]: They changed upstream so I re-generated all of the hashes

**Review by jbedard — APPROVED** [review]

**Review by kormide — APPROVED** [review]

**Comment by gregmagolan** [comment]
`//cypress/tests/module_test:module_test` test target started failing in strange way with this upgrade:

```
==================== Test output for //cypress/tests/module_test:module_test:
Note: You have set the environment variable:

CYPRESS_RUN_BINARY=../../../../cypress_linux-x64/Cypress/Cypress

This overrides the default Cypress binary path used.

[34:1109/022151.364734:ERROR:object_proxy.cc(590)] Failed to call method: org.freedesktop.portal.Settings.Read: object_path= /org/freedesktop/portal/desktop: unknown error type: 

DevTools listening on ws://127.0.0.1:38663/devtools/browser/0d479b79-f6bb-47d0-8273-f4c646a88e1c
Couldn't find tsconfig.json. tsconfig-paths will be skipped

tput: No value for $TERM and no -T specified
```

Not sure what is going on. The test case is passing but cypress exits with non-zero code. @mrmeku Any ideas?

**Comment by jackvincentnz** [comment]
I separated out the checksum upgrade here since I think we should fix that before looking at the version 13 upgrade issue: https://github.com/aspect-build/rules_cypress/pull/58

**Comment by alexeagle** [comment]
I mirrored the latest version.

**Pull request closed without merging**